backend/onedrive/quickxorhash/subtle/xor.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2022 The Go Authors. All rights reserved.
```

Comment:
I'd prefer not to import this source into rclone.

My preference would be to use build tags to make it compile before and after go1.20, so when go1.20 is released, this will get a speed boost.

Reply:
okay, i ll remove that :)  
once go 1.20 release, i’d like to re-open this pr to track it.